src/main/resources/templates/layouts/base-layout.peb

```diff
@@ -0,0 +1,18 @@
+{# @pebvariable name="global" type="org.koil.view.GlobalViewModel" #}
+
+    {% extends 'layouts/base' %}
+    {% block layoutStyle %}
+    <link rel="stylesheet" type="text/css" href="{{  mix("/css/packs/dashboard.css") }}">
```

Comment:
Do we always use the dashboard.css? It feels like this will pull in a lot of extra css that would be for the full dashboard and I'd expect to have a tiny subset of "common" koil css that we could use instead of the full dashboard css.